code/_onclick/hud/_defines.dm

```diff
@@ -23,9 +23,9 @@
 //Middle left indicators
 #define ui_lingchemdisplay "WEST,CENTER-1:15"
 #define ui_lingstingdisplay "WEST:6,CENTER-3:11"
-#define ui_crafting	"12:-10,1:5"
-#define ui_building "12:-10,1:21"
-#define ui_language_menu "11:6,2:-11"
+#define ui_crafting	"EAST-3:-10,1:5"

```

Comment:
Move them to line 81, //Lower right

Reply:
Also they should be on the south border as well.

Reply:
#define ui_crafting	"EAST-4:22,SOUTH:5"
#define ui_building "EAST-4:22,SOUTH:21"
#define ui_language_menu "EAST-4:6,SOUTH:21"
this works.

Reply:
Makes sense to me, updated.